src/network/apis/RoomsApi.ts

```diff
@@ -94,13 +95,12 @@ class RoomsApi extends BaseAPI implements IRoomsApi {
 
 	public deleteRoom(roomId: string): Promise<DeleteRoomResponse> {
 		const meetingId = useStore.getState().rooms[roomId]?.meetingId;
-		return this.fetchAPI(`rooms/${roomId}`, RequestType.DELETE).then(
-			(response: DeleteRoomResponse) => {
-				// Delete the associated meeting
-				if (meetingId) MeetingsApi.deleteMeeting(meetingId);
-				return response;
-			}
-		);
+		const deleteRoom = (): Promise<DeleteMeetingResponse> =>
```

Comment:
Here is better to split and have a simple request deleteRoom and a new method where you call these 2 method separately